website/content/api-docs/secret/pki.mdx

```diff
@@ -16,1178 +16,1109 @@ update your API calls accordingly.
 
-## Read CA Certificate
+## Notice About New Multi-Issuer Functionality
 
-This endpoint retrieves the CA certificate _in raw DER-encoded form_. This is a
-bare endpoint that does not return a standard Vault data structure and cannot
-be read by the Vault CLI; use `/pki/cert` for that. If `/pem` is added to the
-endpoint, the CA certificate is returned in PEM format.
+Vault now allows a single PKI mount to have multiple Certificate Authority
+(CA) certificates ("issuers") in a single mount, for the purpose of
+facilitating rotation. All issuers within a single mount are treated as a
+single authority, meaning that Certificate Revocation List (CRL) configuration
+and other authority URLs are common to all issuers and serial numbers will be
+unique across all issuers. However, since each issuer may have a distinct
+subject and keys, different issuers may have different CRLs.
 
-This is an unauthenticated endpoint.
+It is _strongly_ encouraged to limit the scope of CAs within a mount and not
+to mix different types of CAs (roots and intermediates).
 
-| Method | Path            |
-| :----- | :-------------- |
-| `GET`  | `/pki/ca(/pem)` |
+~> **Note**: Some functionality will not work if a default issuer is not
+   configured. Vault automatically selects the default issuer from the
+   current issuing certificate on migration from an older Vault version.
 
-### Sample Request
+## Accessing Authority Information
 
-```shell-session
-$ curl \
-    http://127.0.0.1:8200/v1/pki/ca/pem
-```
+All consumers of the PKI Secrets Engine mount point will have access to the
+following unauthenticated APIs, useful for reading information about the
+certificate authority in this mount point.
 
-### Sample Response
+This includes information about CA certificates, their chains, and signed
+CRLs, containing an encoded list of revoked certificates previously
+issued by this authority.
 
-```text
-<binary DER-encoded certificate>
-```
+However, two endpoints (for listing all certificates issued by this authority
+and listing all issuers in this mount) are authenticated, though not generally
+considered privileged information.
 
-## Read CA Certificate Chain
+### List Issuers
 
-This endpoint retrieves the CA certificate chain, including the CA _in PEM
-format_. This is a bare endpoint that does not return a standard Vault data
-structure and cannot be read by the Vault CLI; use `/pki/cert` for that.
-Additionally, note that this doesn't include the root authority and so may
-return empty data depending on configuration; use `/pki/cert/ca_chain`'s
-`ca_chain` JSON data field for the entire chain including issuing authority.
+This endpoint returns a list of issuers currently provisioned in this mount.
+The response includes both the issuer's identifier as well as the name chosen
+by the operators; either can be used to refer to the issuer later.
 
-This is an unauthenticated endpoint.
+~> Note: This endpoint is authenticated.
```

Comment:
A bit odd that we are highlighting that the API is authenticated?

Reply:
Open to suggestions here. If you read the header I wrote for this, I said most of these were unauthed (and should be considered public info).

Reply:
Actually the current implementation of the PKI plugin list issuers is not authenticated.